rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -101,14 +101,20 @@ def test_exec(config):
             zone = json.loads(out)
             zone = zone.get("name")
             cmd = 'radosgw-admin zone placement modify --rgw-zone=%s ' \
-                  '--placement-id=default-placement --compression=%s' % (zone,compression_type)
+                  '--placement-id=default-placement --compression=%s' % (zone, compression_type)
             out = utils.exec_shell_cmd(cmd)
+            ceph_version = utils.exec_shell_cmd("ceph version").split()[4]
             try:
                 data = json.loads(out)
-                if data['placement_pools'][0]['val']['storage_classes']['STANDARD']['compression_type'] == compression_type:
-                    log.info('Compression enabled successfully')
+                if ceph_version is 'luminous':
```

Comment:
`is` cannot be used here, since you are doing string comparison. `is` is used to see if python object is same or not . 
look at this

```
In [4]: version = 'rakesh'

In [5]: if verison is 'luminous': 
   ...:     print('entered')
   ...:     
   ...:     
entered
```

Reply:
done